kratos/containers/distributed_csr_matrix.h

```diff
@@ -70,20 +70,25 @@ class DistributedCsrMatrix
     ///@}
     ///@name Life Cycle
     ///@{
-    DistributedCsrMatrix(const DataCommunicator& rComm=ParallelEnvironment::GetDefaultDataCommunicator()) //default constructor. To be used for low level operations
-        : mrComm(rComm)
+
+    //default constructor. To be used for low level operations
+    DistributedCsrMatrix() 
+    {}
```

Comment:
if this is only used for the serialization then I would move it to `private`

Reply:
no, this can also be used in very low level routines, like when converting to amgcl